day01_assignment/5.Operators/question2.js

```diff
@@ -0,0 +1,15 @@
+// Question 2. What do you think would happen if you pass an index beyond the range of the string? 
+// Or if you pass a negative index? Try it out. 
+
+str = "thejoshsoftware"
+
+str_len = str.length
+console.log("length of str",str_len);
+
+console.log(str[-2]);
+console.log(str[16]);
+
+// Output: (It gives undefined for negative and out of range index)
+// 15
+// undefined
+// undefined
```

Comment:
why it gives undefined instead of giving error as we used to see in c and c++ languages?

also what is string in javascript??

give some alternative methods for accessing string characters too and also tell me some method on strings where negative indexing works along with example